dist/actions/entrypoint.sh

```diff
@@ -3,6 +3,16 @@
 
 set -e
 
+# The default backend URL is the Pulumi Managed Service.
+# To use one of the alternate supported backends, set the
+# PULUMI_BACKEND_URL env var according to:
+# https://www.pulumi.com/docs/intro/concepts/state/#to-a-self-managed-backend.
+if [ -n "$PULUMI_BACKEND_URL" ]; then
+    pulumi login -c "$PULUMI_BACKEND_URL"
+else
+    echo "Using Pulumi Managed Service backend"
```

Comment:
Should we instead just always `pulumi login $PULUMI_BACKEND_URL`?

That way we get the real output instead of making something up to echo here.

Reply:
Yep. Even better.